src/tmt_web/generators/html_generator.py

```diff
@@ -4,14 +4,16 @@
 from jinja2 import Environment, FileSystemLoader
 from tmt import Logger, Plan, Test
 
-templ_dir = Path(__file__).resolve().parent / 'templates'
+templ_dir = Path(__file__).resolve().parent / "templates"
```

Comment:
There is a `ruff` setting for that. https://docs.astral.sh/ruff/settings/#lint_flake8-quotes_docstring-quotes

Reply:
That's probably a comment to the #4 or separate PR entirely imho.

Reply:
Ack, let's resolve this separately. I don't think we have any preference about quotes.